hack/cleanup-hypershift-leaks.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I would generally make an image out of this script if we would like to run it as periodic. I would move it to images/ and compose a dockerfile